runner/src/server/services/uploadService.ts

```diff
@@ -226,9 +224,17 @@ export class UploadService {
         )
       ).filter((value) => !!value);
 
+      let pageTitle = page.title;
+      let sectionTitle = page.section.title ?? "";
+
+      if (page.def.metadata.isWelsh) {
```

Comment:
can we not encode both en and cy?

Reply:
Had a discussion on this. It could cause issue with already existing file not being encoded when searching for them on assessment